extension-symbol-descriptor.md

```diff
@@ -1,6 +1,11 @@
-# workspace/xreferences extension to LSP
+# symbolDescriptor extensions to LSP
 
-The `workspace/xreferences` extension to the Language Server Protocol (LSP) enables a language server to export all of the references made from the workspace's code to its dependencies. Additionally, a new `textDocument/xdefinition` method performs the equivalent of `textDocument/definition` while returning metadata about the definition and _optionally_ a concrete location to it.
+Roughly speaking, a `symbolDescriptor` (defined fully below) encodes enough metadata about a symbol to be to uniquely identify it even when it appears in other projects (i.e. as a dependency).
```

Comment:
I would change this to be more accurate, omitting "roughly speaking":

_A SymbolDescriptor contains metadata about a symbol to identify it as uniquely as possible._